### PR TITLE
Simple Emogi Component

### PR DESCRIPTION
**Pull request by TheDoplarEffect**
Component takes a string as a prop (One of the possible Genres) and displays an Emogi in square

### CODE REVIEW

File: src/components/Emogi.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  Fantasy = "🦄",
+  "Science Fiction" = "👽",
+  Thriller = "😱",
+}
+
+export function Emogi({ emojiName }: { emojiName: EmojiLib }) {
+  const emoji = Emojis[emojiName];
+  return (
+    <div className="gap-2 flex flex-col justify-center w-14">
+      <p className="text-3xl flex justify-center items-center w-14 h-14 bg-dark-light rounded-xl">
```
Comment (ReallyBoard123) [inline src/components/Emogi.tsx:42]: text-3xl can actually be predefined in the tailwind config itself.I see that the typography is part of design system. Please update the tailwind config too, and use the appropriate classes from here: https://www.figma.com/file/TOCJzJFMXE7PL1dQyGWqXZ/camp-8-midterm?node-id=193%3A26&t=jUUOOxf7k14aMcI3-1

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
The component looks very good overall. I would also like to see how the component looks like, when you render 4 of them. Is that possible to implement the same in the stories?

**Comment by TheDoplarEffect** [comment]
On the Home page we will map over the different Emogis we want to show using these components

**Comment by ReallyBoard123** [comment]
The other comments are still unresolved?

**Review by ReallyBoard123 — APPROVED** [review]
looks great!

**Pull request merged**